addons/l10n_tr_nilvera_edispatch/templates/l10n_tr_nilvera_edispatch.xml

```diff
@@ -0,0 +1,198 @@
+                <t t-set="is_vkn" t-value="len(partner.vat) == 10"/>
+                <cbc:ID t-att-schemeID="is_vkn and 'VKN' or 'TCKN'" t-out="partner.vat"/>
+            </t>
+        </cac:PartyIdentification>
+        <t t-if="is_vkn">
+            <cac:PartyName>
+                <cbc:Name t-out="partner.name"/>
+            </cac:PartyName>
+        </t>
+        <cac:PostalAddress>
+            <cbc:StreetName t-out="partner.street"/>
+            <cbc:CitySubdivisionName t-out="partner.city"/>
+            <cbc:CityName t-out="partner.state_id.name"/>
+            <cbc:PostalZone t-out="zipcode"/>
+            <t t-set="tr_country" t-value="partner.country_id.with_context(lang='tr_TR')"/>
+            <cac:Country>
+                <cbc:IdentificationCode t-out="partner.country_id.code"/>
+                <cbc:Name t-out="tr_country.name"/>
+            </cac:Country>
+        </cac:PostalAddress>
+        <cac:Contact>
+            <cbc:Telephone t-out="partner.phone"/>
+            <cbc:ElectronicMail t-out="partner.email"/>
+        </cac:Contact>
+        <t t-if="not is_vkn">
+            <t t-set="partner_name" t-value="partner.name.split(' ', 1)"/>
+            <cac:Person>
+                <cbc:FirstName t-out="partner_name[0]"/>
+                <cbc:FamilyName t-out="partner_name[1] if len(partner_name) > 1 else '\u200B'"/>
+            </cac:Person>
+        </t>
+    </template>
+
+    <template id="nilvera_party_template"
+        xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2">
+        <cac:Party>
+            <t t-call="l10n_tr_nilvera_edispatch.nilvera_party_details"/>
+        </cac:Party>
+    </template>
+
+    <template id="nilvera_dispatch_line"
+        xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+        xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+        <cac:DespatchLine>
+            <cbc:ID t-out="line_id"/>
+            <cbc:DeliveredQuantity t-att-unitCode="line.product_uom._get_unece_code()" t-out="line.quantity"/>
+            <cac:OrderLineReference>
+                <cbc:LineID t-out="line_id"/>
+            </cac:OrderLineReference>
+            <cac:Item>
+                <cbc:Name t-out="line.product_id.display_name"/>
+            </cac:Item>
+            <cac:Shipment>
+                <cbc:ID t-out="line_id"/>
+                <cac:GoodsItem>
+                    <cac:InvoiceLine>
+                        <cbc:ID t-out="line_id"/>
+                        <cbc:InvoicedQuantity t-out="line.quantity"/>
+                        <cbc:LineExtensionAmount currencyID="TRY">0</cbc:LineExtensionAmount>
+                        <cac:Item>
+                            <cbc:Name t-out="line.product_id.display_name"/>
+                        </cac:Item>
+                        <cac:Price>
+                            <cbc:PriceAmount currencyID="TRY">0</cbc:PriceAmount>
```

Comment:
Hey, someone got a question about a similar thing in Romania. Is this price supposed to be 0 at all time? and the `LineExtensionAmount` above? or `ValueAmount` below?
Seems dubious

Reply:
So all the amounts are optional?
Good then